uav:
  J:
    J1: 2.3951e-5
    J2: 2.3951e-5
    J3: 3.2347e-5
  m: 4.34
  d: 0.315
  ctf: 0.0008004

controller:
  kx: 16
  kv: 5.6
  kr: 8.81
  kOmega: 2.54

trajectory:
  x0:
    x01: 0
    x02: 0
    x03: 0
  v0:
    v01: 0
    v02: 0
    v03: 0
  R0:
    R0_r1:
      r1_1: 1
      r1_2: 0
      r1_3: 0
    R0_r2:
      r2_1: 0
      r2_2: 1
      r2_3: 0
    R0_r3:
      r3_1: 0
      r3_2: 0
      r3_3: 1
  Omega0:
      Omega01: 0
      Omega02: 0
Omega03: 0